nuget.config

```diff
@@ -2,16 +2,6 @@
 <configuration>

```

Comment:
Similar to no longer needing the action to generate access tokens, this config should no longer need a token to be provided since the CI environment that previously required this token now has access to the artifact feed.